main.js

```diff
@@ -92,12 +107,13 @@ function ftTime(el) {
 
 	if (!date) return;
 
-	var interval = Math.round(((new Date()) - date) / 1000);
-	printer.innerHTML = interval < (365 * 60 * 60 * 24) ? timeAgo(date, interval) : format(date, 'date');
+	printer.innerHTML = ftTime(date);
+
 	el.title = format(date, 'datetime');
 	el.setAttribute('data-o-date-js', '');
 }
 
+
 function timeAgo (date, interval) {
```

Comment:
But your basically not using timeAgo anymore, only the minute and hours bit. You moved the _a day ago_ and the rest to the other function, so if you want it to be separated, you can move your new code into the timeAgo function for example. But you've got the timeAgo function separated into two currently